streamer/src/nonblocking/testing_utilities.rs

```diff
@@ -27,12 +27,15 @@ use {
     solana_tls_utils::{new_dummy_x509_certificate, tls_client_config_builder},
     std::{
         net::{IpAddr, Ipv4Addr, SocketAddr, UdpSocket},
+        num::NonZeroUsize,
         sync::{atomic::AtomicBool, Arc, RwLock},
         time::{Duration, Instant},
     },
     tokio::{task::JoinHandle, time::sleep},
 };
 
+pub(crate) const DEFAULT_NUM_SERVER_THREADS_FOR_TEST: NonZeroUsize = NonZeroUsize::new(8).unwrap();
```

Comment:
With a new good default for the code, we do not need this, as it will be the same '8'